### PR TITLE
e2e: add accountant

### PR DESCRIPTION
**Pull request by djb15**
Rebasing branch to see if it still passes CI

### CODE REVIEW

File: ci_tests/src/accountant.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+// cache the client and signer
+let client: Awaited<ReturnType<typeof getWormchainSigningClient>>;
+let signer: string;
+
+export async function submitAccountantVAA(vaa: Uint8Array) {
+  if (!signer) {
+    // NttAccountantTest = wormhole18s5lynnmx37hq4wlrw9gdn68sg2uxp5rwf5k3u
+    const wallet = await getWallet(
+      "quality vacuum heart guard buzz spike sight swarm shove special gym robust assume sudden deposit grid alcohol choice devote leader tilt noodle tide penalty"
```
Comment (djb15) [inline ci_tests/src/accountant.ts:16]: I'm going to assume this seed phrase is fine being out in the wild?

**Review by djb15 — COMMENTED** [review]
I can't approve as I opened the PR but lgtm

File: ci_tests/src/accountant.ts (hunk 1 above)
Comment (evan-gray) [reply ci_tests/src/accountant.ts:16]: Yes it’s a default cosmos one and in the WH repo

File: ci_tests/.gitignore [hunk 2]
```diff
@@ -1,2 +1 @@
 evm_binding
-solana_binding
```
Comment (evan-gray) [inline ci_tests/.gitignore:2]: should this have been removed? there still should be nothing committed here.

File: ci_tests/.dockerignore [hunk 3]
```diff
@@ -1,3 +1,2 @@
 **/node_modules
 evm_binding
-solana_binding
```
Comment (evan-gray) [inline ci_tests/.dockerignore:3]: these are / should be copied from the `solana/Dockerfile`. why has this been removed?

**Review by evan-gray — CHANGES_REQUESTED** [review]
reviewed Tilt and ci_tests related changes

**Review by evan-gray — APPROVED** [review]
reviewed tilt and ci_tests

**Review by nik-suri — APPROVED** [review]

**Pull request merged**